src/browser.ts

```diff
@@ -115,13 +135,24 @@ export class GrpcClient {
         metadata,
         callback
       ) => {
-        languageServiceStub[methodName].apply(languageServiceStub, [
-          req,
-          callback,
-        ]);
+        const cancelHandler = languageServiceStub[methodName].apply(
+          languageServiceStub,
+          [req, callback]
+        ) as CancelHandler;
+        return {
+          cancel: () => {
+            if (cancelHandler.canceller === undefined) {
+              console.warn(
+                'AbortController not found: Cancellation is not supported in this environment'
+              );
+            } else {
```

Comment:
This is a personal style nit: I would put a `return;` after the warning in the `if` statement, and remove this `else` statement. Thus, if it goes into the if condition, the code stops there. If we don't go in, it falls through to the abort

Reply:
Agreed. That will move the rest of the functions one indentation level left which is always good.

Reply:
Sure, sounds good.

Done.